build_tools/docker/rbe_toolchain/Dockerfile

```diff
@@ -21,23 +21,29 @@
 
 FROM gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:1a8ed713f40267bb51fe17de012fa631a20c52df818ccb317aaed2ee068dfc61
 
-RUN apt-get update
-RUN apt-get install -y python3 python3-pip
-RUN python3 -m pip install --upgrade pip
-RUN python3 -m pip install numpy
+RUN apt-get update \
+    && apt-get install -y \
+        python3 \
+        python3-pip \
+    && python3 -m pip install --upgrade pip \
+    && python3 -m pip install numpy
+
+# Dependency for python3.6-dev. Needs to be installed separately from the above
+# for... some reason
+RUN apt-get update && apt-get install -y software-properties-common
 
-# Install dependencies for python3.6-dev
-RUN apt-get install -y software-properties-common
 # apt-add-repository requires a version of python with the softwareproperties
 # module. To use this command, we:
 #   1. remove the symlink to python3 from python3.6 and symlink it to python3.5
 #   2. run apt-add-repository with python3 = python3.5
 #   3. resymlink python3 to /opt/python3.6/bin/python3.6
 # See https://github.com/google/iree/issues/1966 for more information.
-RUN rm /usr/bin/python3 && ln -s /usr/bin/python3.5 /usr/bin/python3
-RUN add-apt-repository ppa:deadsnakes/ppa
-RUN rm /usr/bin/python3 && ln -s /opt/python3.6/bin/python3.6 /usr/bin/python3
+RUN rm /usr/bin/python3 \
+    && ln -s /usr/bin/python3.5 /usr/bin/python3 \
```

Comment:
Could use `update-alternatives`. But I don't care that much.

Reply:
Yeah we can fiddle with this more. Phoenix played with it quite a bit, I think. I'd like to get this change in and then we can try fiddling more.

Reply:
@phoenix-meadowlark want to try update-alternatives here? Is that something you already tried?